devtools/create_achievements/create_achievements.sh

```diff
@@ -0,0 +1,87 @@
+python steam_achievements.py -v --steamid 338420
+python steam_achievements.py -v --steamid 357490
+python steam_achievements.py -v --steamid 364350
+python steam_achievements.py -v --steamid 364390
+python steam_achievements.py -v --steamid 367110
+python steam_achievements.py -v --steamid 370910
+python steam_achievements.py -v --steamid 370910 --saveasgalaxyid 1460710709
+python steam_achievements.py -v --steamid 396090
+python steam_achievements.py -v --steamid 420180
+python steam_achievements.py -v --steamid 425600
+python steam_achievements.py -v --steamid 425600 --saveasgalaxyid 1444830704
+python steam_achievements.py -v --steamid 439310
+python steam_achievements.py -v --steamid 468530
+python steam_achievements.py -v --steamid 509920
+python steam_achievements.py -v --steamid 551840
+python steam_achievements.py -v --steamid 556060
+python steam_achievements.py -v --steamid 556060 --saveasgalaxyid 1745746005
+python steam_achievements.py -v --steamid 561770
+python steam_achievements.py -v --steamid 595560
+python steam_achievements.py -v --steamid 603870
+python steam_achievements.py -v --steamid 610900
+python steam_achievements.py -v --steamid 631570
+python steam_achievements.py -v --steamid 631570 --saveasgalaxyid 1845001352
+python steam_achievements.py -v --steamid 654550
+python steam_achievements.py -v --steamid 673850
+python steam_achievements.py -v --steamid 761460
+python steam_achievements.py -v --steamid 782280
+python steam_achievements.py -v --steamid 872750
+python steam_achievements.py -v --steamid 872750 --saveasgalaxyid 1455980545
+python steam_achievements.py -v --steamid 904750
+python steam_achievements.py -v --steamid 1028740
+python steam_achievements.py -v --steamid 1133950
+python steam_achievements.py -v --steamid 1142230
+python steam_achievements.py -v --steamid 1147030
+python steam_achievements.py -v --steamid 1181570
+python steam_achievements.py -v --steamid 1220930
+python steam_achievements.py -v --steamid 1251910
+python steam_achievements.py -v --steamid 1270590
+
+#TWINE games:
+python steam_achievements.py -v --steamid 397330
+
+#WME games:
+python steam_achievements.py -v --steamid 257690
+python steam_achievements.py -v --steamid 257690 --saveasgalaxyid 1457085654
+python steam_achievements.py -v --steamid 270570
+python steam_achievements.py -v --steamid 281060
+python steam_achievements.py -v --steamid 281080
+python steam_achievements.py -v --steamid 286320
+python steam_achievements.py -v --steamid 286360
+python steam_achievements.py -v --steamid 378630
+python steam_achievements.py -v --steamid 291710
+python steam_achievements.py -v --steamid 337130
+python steam_achievements.py -v --steamid 340370
+python steam_achievements.py -v --steamid 349140
+python steam_achievements.py -v --steamid 405780
+python steam_achievements.py -v --steamid 574420
+python steam_achievements.py -v --steamid 1064660
+
+zip -9j achievements.dat gen/* static/*
+mv -vf achievements.dat ../../dists/engine-data
```

Comment:
Missing newline. 

I'm not sure about the guidelines here - but we should maybe check for 7zip being available. It produces smaller files. But this is most likely something for a new PR - as this might touch more locations.

Reply:
Newline fixed.

Thanks for pointing at 7zip. I have added a TODO to remember about it at 5f5f63777760fef5d920b4d0c206896aceea6f9b
Also, i added `touch --date` and `zip -X` to get the same dat file on different runs.